File: Problemas/400-499/427. Yo soy tu/java/Problema427.java
import java.util.Scanner;

public class Problema427 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int	numCasos = sc.nextInt();
		String nombre;
		String parentesco;
		
		sc.nextLine();
		for (int i=0;i < numCasos;i++) {
			nombre = sc.nextLine();
			parentesco = sc.nextLine();
			
			if (nombre.equals("Luke") && parentesco.equals("padre")) {
				System.out.println("TOP SECRET");
			}
			else {
				System.out.println(nombre + ", yo soy tu " + parentesco);
			}
		}
	}
}

File: Problemas/400-499/427. Yo soy tu/c/Problema427.c
#include <stdio.h>
#include <string.h>

int main(int argc, char const *argv[])
{
		char			nombre[101];
		char			parentesco[101];
		unsigned	int	casos;
		unsigned	int	i;
		
		scanf("%d", &casos);

		for (i = 0; i < casos; i++) {
			scanf("%s", nombre);
			scanf("%s", parentesco);

			if (strcmp(nombre, "Luke") == 0 && strcmp(parentesco, "padre") == 0)
				printf("TOP SECRET\n");
			else
				printf("%s, yo soy tu %s\n", nombre, parentesco);
		}

		return 0;
}